src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
+  background: black;
+}
+
+.block {
+  box-sizing: border-box;
+  width: 350px;
+  padding: 20px;
+  margin: 20px 20px 40px;
+}
+
+.block-yellow {
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.